js/src/forum/components/Search.js

```diff
@@ -177,6 +177,13 @@ export default class Search extends Component {
           .one('mouseup', (e) => e.preventDefault())
           .select();
       });
+
+    // Since extensions might add elements above the search box on mobile,
+    // we need to calculate and set the max height dynamically.
+    const resultsElementMargin = 14;
+    const maxHeight =
+      window.innerHeight - this.element.querySelector('.Search-input>.FormControl').getBoundingClientRect().bottom - resultsElementMargin;
+    this.element.querySelector('.Search-results').style['max-height'] = `${maxHeight}px`;
```

Comment:
since we're adding it in `oncreate`, the max height is set early, but what happens when the keyboard shows up ? I'm assuming the value won't match then and so the issue will still persist, I can't test this but onupdate seems a better place.